### PR TITLE
Create SideBar by Adrian Serzysko 😎 

### PR DESCRIPTION
**Pull request by adrianserzy**

### CODE REVIEW

File: app.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
-  <div>
-    <SideBar/>
-    <NuxtPage/>
+  <div class="flex">
+    <SideBar />
+    <div class="h-full w-[300px] bg-white"></div>
```
Comment (lobarzanek) [inline app.vue]: Please explain the functionality of this div:
`<div class="h-full w-[300px] bg-white"></div>`

File: components/Side/Bar.vue [hunk 2]
```diff
@@ -1,4 +1,216 @@
...
+    },
+  },
+};
+</script>
 <template>
-    <div>
+  <div class="h-screen w-[300px] flex flex-col items-center bg-white">
+    <h1 class="text-[26px] text-[#333] p-[20px]">Dashboard</h1>
+    <div class="mt-[30px]">
+      <div :class="$style.link" class="group duration-500">
```
Comment (lobarzanek) [inline components/Side/Bar.vue]: `<div :class="$style.link" class="group duration-500"> ... </div>` should be wrapped with:
`<NuxtLink to="/"> ... </NuxtLink>` (where "/" is a path to a page)
and declared as reusable component with example properties: svg, text, path etc.
SVG's should be imported from dedicated file (maybe assests/SVG/SvgName ?)

Components can be created based on declared objects array with `v-for` https://vuejs.org/api/built-in-directives.html#v-for

File: components/Side/Bar.vue [hunk 3]
```diff
@@ -1,4 +1,216 @@
...
+            ></path>
+            <path
+              d="M2 12C2 7.28595 2 4.92893 3.46447 3.46447C4.92893 2 7.28595 2 12 2C16.714 2 19.0711 2 20.5355 3.46447C22 4.92893 22 7.28595 22 12C22 16.714 22 19.0711 20.5355 20.5355C19.0711 22 16.714 22 12 22C7.28595 22 4.92893 22 3.46447 20.5355C2 19.0711 2 16.714 2 12Z"
+              stroke="#333"
+              stroke-width="1.5"
+              class="group-hover:stroke-white"
+            ></path>
+          </g>
+        </svg>
+        <p :class="$style.text">Dashboard</p>
```
Comment (lobarzanek) [inline components/Side/Bar.vue]: For dashboard you could use more explicit icon, ie: https://www.svgrepo.com/svg/520684/dashboard

**Review by lobarzanek — CHANGES_REQUESTED** [review]
Except changes from review please pull current main branch version and add mobile view

File: app.vue (hunk 1 above)
Comment (adrianserzy) [reply app.vue]: I checked one thing - the div will be removed on the next push

**Review by lobarzanek — APPROVED** [review]

**Review by kamilziemann — APPROVED** [review]

**Pull request merged**